scripts/run_redis.sh

```diff
@@ -0,0 +1,5 @@
+set -e
+
+docker rm -f redis || true
+export DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
maybe let's move `run_redis.sh` and redis config from node-cli and skale-admin to the helper-scripts?